pkg/apis/operator/v1alpha1/types_extension.go

```diff
@@ -142,4 +142,11 @@ const (
 	ExtensionRequiredRuntime gardencorev1beta1.ConditionType = "RequiredRuntime"
 	// ExtensionRequiredVirtual is a condition type for indicating whether the extension is required in the virtual garden cluster.
 	ExtensionRequiredVirtual gardencorev1beta1.ConditionType = "RequiredVirtual"
+
+	// ControllerInstallationsHealthy is a constant for a condition type indicating the health of the controller installations.
+	ControllerInstallationsHealthy = "ControllerInstallationsHealthy"
+	// ExtensionHealthy is a constant for a condition type indicating the extension's health.
+	ExtensionHealthy gardencorev1beta1.ConditionType = "Healthy"
```

Comment:
```suggestion
	ExtensionHealthy gardencorev1beta1.ConditionType = "RuntimeHealthy"
```

Does that make sense for clarity? My previous suggestion was based on that we go with a single `Healthy` condition. My apologies 😄